### ISSUE TITLE
OC API problems

### ISSUE THREAD

**Issue by Tahakk** [ZeroNoRyouki/BigReactors#146]
Hi, not even going to try guessing what changed till 0.4.5.4x versions, but basically from wha I can get there is some kind of problem with OC API and that makes some of the programs unusable. Since I didn't saw any reaction on Twitter I'm going to post it here:

More info can be found here:
https://oc.cil.li/index.php?/topic/918-big-reactors-grid-control/&do=findComment&comment=6534

I'm not the guy who wrote that program, just user that have problems that others describe there.

**Comment by ZeroNoRyouki** [comment]
I'm not making the computer yield, at least not intentionally

The big change in the OC interface was that I was forced to move to an Environment/capabilities based solution so there could be something there

I've tested the OC interface with Krakaen's control program and it worked fine (https://www.youtube.com/watch?v=8IycdrAkHE8)

**Comment by Tahakk** [comment]
Well, now we wait on XyFreak's response in hope this get resolved somehow. Thanks for answer tho.

**Comment by ResqDiver1317** [comment]
Going to watch this issue myself as I'm having the same issue with the OC program being EXTREMELY slow.

**Comment by ZeroNoRyouki** [comment]
I'm closing this in favor of #152
